src/main/java/com/rsr5/gocd/riemann_notifier/DuplicateToArrayJsonNodeDeserializer.java

```diff
@@ -0,0 +1,33 @@
+package com.rsr5.gocd.riemann_notifier;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.deser.std.JsonNodeDeserializer;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+public class DuplicateToArrayJsonNodeDeserializer extends JsonNodeDeserializer {
+    @Override
+    protected void _handleDuplicateField(
+            JsonParser p,
+            DeserializationContext ctxt,
+            JsonNodeFactory nodeFactory,
+            String fieldName,
+            ObjectNode objectNode,
+            JsonNode oldValue,
+            JsonNode newValue) throws JsonProcessingException {
```

Comment:
Minor Java style point, but since you're breaking all the args onto new lines it'd be more consistent to also break before the thrown exception list:
```suggestion
            JsonNode newValue)
            throws JsonProcessingException {
```